modules/rules/main.tf

```diff
@@ -61,7 +61,7 @@ resource "google_storage_bucket_object" "main" {
   count   = local.rules_count
   name    = element(local.files, count.index)
   content = element(data.template_file.main.*.rendered, count.index)
-  bucket  = var.bucket
+  bucket  = var.server_gcs_module.forseti-server-storage-bucket
 
   lifecycle {
```

Comment:
Just curious- do we need to provide the `folder_id` [here ](https://github.com/forseti-security/terraform-google-forseti/blob/c73c874f833afb2759db08e09271707332b5c50f/modules/rules/main.tf#L54)as well if there are users who do not use at the org level?

Reply:
Can you please help me understand a little better?  Something along the lines of
```hcl
name = "${var.folder_id}/element(local.files, count.index)"
```

Reply:
I guess this is more for users who are just using the scanner files out of the box- for example [this](https://github.com/forseti-security/terraform-google-forseti/blob/c73c874f833afb2759db08e09271707332b5c50f/modules/rules/templates/rules/bigquery_rules.yaml) scanner file/template uses the ${org_id} variable that you input- if they do not provide an `org_id` variable, the values would just be `""` and would not necessarily be helpful for users who are only at the folder level, for example, so they would need to probably go in and manually change.

Should be ok though..

Reply:
I understand what you are suggestion.  Thanks for the clarification.  I think we do need to add it.

Reply:
Though none of the rule templates as written have a folder_id.  On second thought, let's leave it out for the moment.